### PR TITLE
added blog on user authentication with python

### PR DESCRIPTION
**Pull request by LordGhostX**
description:

#### Please note that in case of below checklist not updated accordingly, the maintainers have the right to immediately close your pull request.

- [x] You created a branch from `dev` branch.
- [x] You PR is raised on `dev` branch and not on`master`.
- [x] Blog follows [Blog Guidelines](https://github.com/LoginRadius/engineering-portal/blob/master/GUIDELINES.md).
- [x] You have read the [Contribution Guidlines](https://github.com/LoginRadius/engineering-portal/blob/master/CONTRIBUTING.md) before creating this PR.
- [x] Checked for any duplicate blog entry.
- [x] Checked for all grammatical errors in the blog.
- [x] You have not published the blog content anywhere else on the internet on platforms like medium, dev.to etc.
- [x] The content is not copy-pasted from anywhere else on the Internet.

##### Author Details for Blog Post Entry

id: Solomon Esenyi
bio: Python Developer and Technical Writer with a passion for open-source, cryptography, and serverless technologies.
github: LordGhostX
linkedin: solomon-esenyi
twitter: LordGhostX

### CODE REVIEW

**Comment by LordGhostX** [comment]
@puneetsingh24 @pareek-narendra Is there anything I need to do before its reviewed and merged?

**Review by pareek-narendra — CHANGES_REQUESTED** [review]
@LordGhostX 
Thanks for writing valuable content. Please make these minor changes

- Add a description of the blog in the index.md file
- Use the valid link under the What Is LoginRadius? heading (https://accounts.loginradius.com/auth.aspx?action=register&return_url=https://dashboard.loginradius.com/login)
- Use the valid link under the Acquiring LoginRadius API Credentials heading (https://accounts.loginradius.com/auth.aspx?return_url=https://dashboard.loginradius.com/login)
- Add author info under content->author.yaml file

File: content/user-authentication-in-python/index.md [hunk 1]
```diff
@@ -0,0 +1,274 @@
+---
+title: Implementing User Authentication in a Python Application
+date: "2021-06-29"
+coverImage: "coverImage.jpg"
+author: "Solomon Esenyi"
+tags: ["Python", "Authentication", "Flask"]
+---
```
Comment (pareek-narendra) [inline content/user-authentication-in-python/index.md:7]: Add 1-2 liner description for your blog

For example
description: ""

Comment (LordGhostX) [reply content/user-authentication-in-python/index.md:8]: Thank you, I've made the necessary changes

**Comment by LordGhostX** [comment]
Thank you @pareek-narendra 

I've made all requested changes

File: content/user-authentication-in-python/index.md [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+
+Every web application built with Python at one point or another would need to implement user authentication features. This article will cover implementing authentication and proper handling of user identity information using [LoginRadius](https://loginradius.com) and Flask.
+
+## Getting Started with LoginRadius
+
+### What Is LoginRadius?
+
+[LoginRadius](https://accounts.loginradius.com/auth.aspx?action=register&return_url=https://dashboard.loginradius.com/login) is a cloud-based consumer identity and access management (CIAM) platform that allows seamless user authentication and SSO integration into your application. LoginRadius is simple to use, completely secure, and highly customizable.
+
+To proceed with this tutorial, you will need an account with LoginRadius. If you have not created one before now, create one on the [LoginRadius website](https://accounts.loginradius.com/auth.aspx?action%3Dregister).
```
[the remark this answers is not in the document]
Comment (LordGhostX) [reply content/user-authentication-in-python/index.md]: Thank you for pointing that out, I've fixed it now

**Review by pareek-narendra — APPROVED** [review]

**Pull request merged**